Sources/MapboxMaps/Gestures/GestureHandlerDelegate.swift

```diff
@@ -28,7 +28,7 @@ internal protocol GestureHandlerDelegate: AnyObject {
     func pinchChanged(with zoom: CGFloat, anchor: CGPoint, previousAnchor: CGPoint)
 
     // Pinch has completed with a final scale and possible drift
-    func pinchEnded(with finalScale: CGFloat, andDrift possibleDrift: Bool, andAnchor anchor: CGPoint)
+    func pinchEnded(with finalScale: CGFloat, andAnchor anchor: CGPoint)
```

Comment:
We should update the comment if we are removing this drift var. 

Which to that point, why are we removing the drift param

Reply:
Good Call. I removed drift because I found that it isn't actually used downstream (in the GestureManager).

Reply:
Nice! just to make sure, I want to tag in @nishant-karajgikar is there any impact with removing this drift variable? I assume not since it's not being used

Reply:
I can definitely add a new ticket for follow-up work but I'm not totally clear what the expected behavior is. Some clarification on that would be helpful too.